config/config.go

```diff
@@ -28,7 +28,7 @@ import (
 )
 
 var (
-	patJobName    = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_-]*$`)
+	patJobName    = regexp.MustCompile(`^[a-zA-Z0-9_][a-zA-Z0-9_-]*$`)
```

Comment:
Oh no... sorry, it was just my internet connection going bad again (https://twitter.com/juliusvolz/status/776725728436031488) and the page not updating properly :-/

All good.

(fun fact, took me many retries to post this comment)